README.md

```diff
@@ -290,9 +286,21 @@ $posts = PostFactory::randomSet(4); // array containing 4 "Post|Proxy" objects
 $posts = PostFactory::randomRange(0, 5); // array containing 0-5 "Post|Proxy" objects
 ```
 
-**WARNING**: Never instantiate your `ModelFactory` with the constructor (ie `new PostFactory()`). This will
+### Instantiate your `ModelFactory`
+
+One should never instantiate your `ModelFactory` with the constructor (ie `new PostFactory()`). This will
 cause the factory to not be instantiated properly. Always instantiate with `PostFactory::new()`.
 
+The first argument to `PostFactory::new()` will allow you to overwrite the default
+values that are defined in the `PostFactory::getDefaults()`.
+
+```php
+use App\Factory\PostFactory;
+
+PostFactory::new(['title' => 'My Title'])->create();
+PostFactory::new(['title' => 'My Title'])->createMany(5);
```

Comment:
> Is this clear enough? Should I expand on my example by doing $postFactory = PostFactory::new([...]);?

Here you mean? Yeah, I think that might in understanding that all factories created will have this new default.

Reply:
Yes. Exactly.

Reply:
Updated!